test/test_linalg.py

```diff
@@ -7764,6 +7764,28 @@ def test_tensordot(self, device):
         an = torch.from_numpy(np.tensordot(np.zeros((), dtype=np.float32), np.zeros((), dtype=np.float32), 0))
         self.assertEqual(a, an)
 
+    def test_linalg_trace(self, device):
+        inputs = [
+            {'shape': (1, 1), 'offsets': [0]},
+            {'shape': (10, 1), 'offsets': [0, -9]},
```

Comment:
What happens if offset is an absurd number, like 100?

Reply:
`RuntimeError` will be raised if `offset` is out of range. I will add a test for this case!

Reply:
> `RuntimeError` will be raised if `offset` is out of range. I will add a test for this case!

Make it an ErrorInput